examples/train_model.py

```diff
@@ -56,6 +56,10 @@ def setup_args():
     train.add_argument('-vmt', '--validation-metric', default='accuracy',
                        help='key into report table for selecting best '
                             'validation')
+    train.add_argument('-vcut', '--validation-cutoff',
```

Comment:
default should be for >=1?

Reply:
sure if you prefer! this was 0.995 below before ("rounds up to 1") but I'll change to 1